blog post for oct 22 update (#2)
openmls/blog: content/2022-10-19-update.md

---
title: "OpenMLS Update"
date: 2022-10-19
tags: ["mls", "openmls", "roadmap"]
notoc: true
author: "Franziskus Kiefer"
author_link: "https://www.franziskuskiefer.de"
---

_After a few months of silence, we are picking up work on OpenMLS again! 🎉_

> [Phoenix R&D] and [Cryspen] joined forces to bring OpenMLS closer to v1.0.

Reviewer: ```suggestion
[Phoenix R&D] and [Cryspen] join forces to bring OpenMLS closer to v1.0.
```

The [MLS protocol] draft is in the IETF working group's [last call] and is thus
on track to become an RFC soon.
We want to make sure that OpenMLS is ready for wider adoption by the time the
RFC is ready.
To achieve this we start by catching up with all the changes to the protocol
(up to the current draft-16), improving test coverage of the code, and working
towards a more comprehensive test framework for interoperability.

> We want to thank the [Sovereign Tech Fund] that is [funding OpenMLS] development
> as one of its first projects.

Reviewer: ```suggestion
We want to thank the [Sovereign Tech Fund] for [funding OpenMLS]
as the first project in its pilot round.
```
